modules/virt-about-top-consumers-projects-vms-nodes.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assembly:
+//
+// * virt/logging_events_monitoring/virt-reviewing-virtualization-overview.adoc
+
+[id="virt-about-reviewing-top-consumers-projects-vms-nodes_{context}"]
+= About top consumers for projects, virtual machines, and nodes
```

Comment:
I am hesitant to change the title for the module. My general thought is that reference modules offer optional type of content. This concept module contains information that is useful to the user BEFORE they perform a procedure (they learn about each resource and can determine which resource they want to view when they perform the procedure). I see this module as a precursor to the procedure module that comes right after it. Perhaps I could reword the intro to the table as follows: The follow table provides a description of resources monitored for top consumers. Let me know your thoughts.

Reply:
Upon going back and reviewing this module, it seems to fit more into a reference pattern as you noted - especially because it doesn't precede any tasks or actions to be taken. Changing this to a reference module.